ddl/table.go

```diff
@@ -896,3 +906,48 @@ func checkAddPartitionValue(meta *model.TableInfo, part *model.PartitionInfo) er
 	}
 	return nil
 }
+
+func onRepairTable(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64, _ error) {
```

Comment:
If we use `admin repair`, how the binlogs will be?

Reply:
Should be exactly same with other ddl job, cause binlog use job.ID and job.query to write.